data_structures/binary_tree/binary_search_tree.py

```diff
@@ -155,6 +155,15 @@ def find_kth_smallest(self, k: int, node: Node) -> int:
         self.inorder(arr, node)  # append all values to list using inorder traversal
         return arr[k - 1]
 
+    def total_possible_BST(self, node:Node) -> int:
+        from math import factorial
+        arr = []
+        self.inorder(arr, node)
+        n = len(arr)
+        # Catalan number Cn = (2n)! / ((n + 1)! * n!)
+        return factorial(2 * n) // factorial(n + 1) // factorial(n)
+
```

Comment:
```suggestion
    def total_possible_binary_search_trees(self, node:Node) -> int:
        arr = []
        self.inorder(arr, node)
        n = len(arr)
        # Catalan number Cn = (2n)! / ((n + 1)! * n!)
        return factorial(2 * n) // factorial(n + 1) // factorial(n)
```

Reply:
Please move the import to the top of the file.

Reply:
okay